tests/index.test.html

```diff
@@ -124,16 +130,19 @@
             expect(server.requests[0].url).to.be.equal(url);
           });
 
-          it("skip everything because Read the Docs Embed is present", () => {
+          // The last call of "done" function is required here because is how Mocha works with async code.
+          // Read more at https://mochajs.org/#asynchronous-code
+          it("skip everything because Read the Docs Embed is present", function (done) {
```

Comment:
Does this not work with arrow functions?

```suggestion
          it("skip everything because Read the Docs Embed is present", (done) => {
```